app/views/admins/weekly_user_stats.mobile.haml

```diff
@@ -0,0 +1,14 @@
+.col-md-9
+  %h2
+    = t(".current_server", date: Time.now.to_date)
```

Comment:
If [I understand correctly](https://github.com/diaspora/diaspora_federation/blob/develop/Changelog.md#refactor-1) ``Time.now.iso8601`` is the correct answer?!